app/src/main/java/com/cornellappdev/android/score/data/models/Team.kt

```diff
@@ -0,0 +1,39 @@
+package com.cornellappdev.android.score.data.models
+
+data class Team(
+    val sport: Sport,
+    val genderDivision: GenderDivision
+)
```

Comment:
Wait maybe I'm dumb, but what does this class represent? Did I ask you to make this 😅

Reply:
NAURR sorry justin i misinterpreted